src/locutus_util/dispatcher.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+from locutus_util.ontology_api_etl import ontology_api_etl
+from locutus_util.seed_data_etl import seed_data_etl
+from locutus_util.common import (
+    FETCH_AND_UPLOAD, UPLOAD_FROM_CSV, UPDATE_CSV,
+    UPDATE_ONTOLOGY_API, UPDATE_SEED_DATA
+)
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Manage data ETL of data into Firestore."
+    )
+    parser.add_argument(
+        '-p', '--project', 
+        required=True, 
+        help="GCP Project to edit."
+    )
+    parser.add_argument(
+        '-o', '--option', 
+        choices=[UPDATE_ONTOLOGY_API, UPDATE_SEED_DATA],
+        required=True,
+        help=(
+            f"{UPDATE_ONTOLOGY_API}: Update Ontology API data.\n"
+            f"{UPDATE_SEED_DATA}: Update Seed Data."
+        )
+    )
```

Comment:
Let's work in some reasonable defaults for these (probably most if not all of the pieces other than the project)

Reply:
I don't see either of these being a priority/more likely to be run. Maybe I could set something up that would run both scripts? I was thinking we decided ontology_api was not seed data.